lib/run-service.js

```diff
@@ -0,0 +1,53 @@
+const {exec} = require('child_process')
+const logger = require('electron-timber')
+const {app} = require('./app')
+
+const runServiceLogger = logger.create({name: 'RunService'})
+
+runServiceLogger.log('Run Service is awake')
+
+const {
+  SERVICE_NAME,
+  SERVICE_PATH,
+  SERVICE_PORT
+} = process.env
+
+const {
+  paths: {
+    nvs,
+    editor,
+    logs
+  }
+} = app
+
+const command = `
+  . "${nvs}/nvs.sh" && \\

```

Comment:
did we previously use `nvs`? also is this getting installed or did you assume the user would have it which that may not?

Reply:
Yes, it was always there

It gets installed as part of the first run setup